api_v3/lib/types/filters/base/KalturaUserEntryBaseFilter.php

```diff
@@ -74,7 +74,7 @@ public function getOrderByMap()
 	public $entryIdNotIn;
 
 	/**
-	 * @var int
+	 * @var string
```

Comment:
Please note that although there is still a change it was auto-generated because there was a change in KalturaUserEntry. The rest of the code was moved elsewhere